### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pycaret.internal.preprocess.transformers'

### DISCUSSION THREAD

**Discussion by JSPAnthony** [pycaret/pycaret#2873] [Q&A]
Dear All,

I built a model using Pycaret random classifier, however when I try to deploy the model, the following error pops up from the pickle.py file,
"__import__(module, level=0)
ModuleNotFoundError: No module named 'pycaret.internal.preprocess.transformers'; 'pycaret.internal.preprocess' is not a package."

The Pycaret version is 2.3.10 and my Python version is 3.8.8.

What could be the problem since the pickle.py file is system file?

Thank you for your support

**Comment by celestinoxp** [comment]
@ngupta23

**Reply by JSPAnthony** [reply]
Hi Celetinoxp, I went throogh @ngupta23 comments but didn't see any relation to my issue.

**Reply by celestinoxp** [reply]
> Hi Celetinoxp, I went throogh @ngupta23 comments but didn't see any relation to my issue.

i'm not comfortable with pycaret 2.3.10.
I'm focused on pycaret 3.0RC. So I just entered ngupta for him to receive a notification of this post, as he is more comfortable than I am in this matter.
I think he's a little busy, so please wait a bit. Thanks

**Reply by ngupta23** [reply]
There is not enough information here to respond. Please file a new Bug report (instead of Discussion) with the reproducible example and the complete error stack.

**Reply by JSPAnthony** [reply]
Noted

**Comment by JSPAnthony** [comment]
Noted with thanks
